crates/network-scanner/src/interfaces/mod.rs

```diff
@@ -72,10 +88,13 @@ pub struct InterfaceAddress {
 
 #[derive(Debug, Clone)]
 pub struct NetworkInterface {
+    // Linux interfaces does not come with an id
+    pub id: Option<String>,
```

Comment:
question: Should it be named `adapter_id` or `adapter_name` instead? Or is this naming used somewhere else?